examples/hf_trainer_api/hf_image_classification/deepspeed.yaml

```diff
@@ -1,12 +1,13 @@
 name: vision transformer deepspeed stage 1
 debug: false
 environment:
-    environment_variables:
-        - NCCL_DEBUG=INFO
-        # You may need to modify this to match your network configuration.
-        - NCCL_SOCKET_IFNAME=ens,eth,ib
-    image:
-      gpu: determinedai/environments:cuda-11.3-pytorch-1.10-deepspeed-0.8.3-gpu-0.21.2
+  environment_variables:
+    - NCCL_DEBUG=INFO
+    # You may need to modify this to match your network configuration.
+    - NCCL_SOCKET_IFNAME=ens,eth,ib
+  image:
+    gpu: determinedai/environments:cuda-11.3-pytorch-1.10-deepspeed-0.8.3-gpu-6eceaca
```

Comment:
Why do we need to specify both, cpu and gpu images?

Reply:
Don't believe we do anymore. At one point we needed to because the search runner experiment (which runs on cpu) had some dependency on `deepspeed`.